frontend/src/components/Secrets/GSecretDialogGcp.vue

```diff
@@ -121,13 +122,19 @@ export default {
     'update:modelValue',
   ],
   setup () {
+    const { secretStringDataRefs } = useProvideCredentialContext()
+
+    const { serviceAccountKey } = secretStringDataRefs({
+      'serviceaccount.json': 'serviceAccountKey',
```

Comment:
the service account input field should be hidden by default (especially when updating a gcp secret)

Reply:
This is not possible for textarea

Reply:
Added custom css classes to make it work